s3-eventbridge-glue/example-pattern.json

```diff
@@ -0,0 +1,63 @@
+{
+    "title": "Automatic Data Transformation and Catalog with Amazon S3 notifications and AWS Step functions",
+    "description": "This pattern facilitates the cataloging and transformation of CSV data, using an event-driven architecture to automatically trigger data cleaning and transformation jobs",
```

Comment:
`description`: description does not meet maximum length of 150

Reply:
Resolved